lib/vscode_helper.dart

```diff
@@ -68,7 +68,7 @@ void updateLaunchJsonFile(FlutterProject project, Uri observatoryUri) {
     config['cwd'] = project.hasExampleApp
         ? r'${workspaceFolder}/example'
         : r'${workspaceFolder}';
-    config['observatoryUri'] = observatoryUri.toString();
+    config['vmServiceUri'] = observatoryUri.toString();
```

Comment:
`"observatoryUri"` is deprecated.

Reply:
This term is still used in the document(such as `The observatory URL`, `--observatory-uri`). is it okay?

Reply:
What I meant was that the config property had been deprecated by the VS Code extension.

![image](https://user-images.githubusercontent.com/40190588/188627640-18a4d6e1-121d-4092-a45f-8e51d002d665.png)

https://github.com/flutter/devtools/issues/1642

The **Dart extension** uses the term "VM service", but the term "observatory" still also seems valid across the **Flutter SDK** (the code and documentation). I'm not sure about DevTools' case but anyway I'm trying to be consistent with the Flutter SDK as much as possible.